VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/tests/waiting-room-base.component.eventhub-events.spec.ts

```diff
@@ -383,7 +383,7 @@ describe('WaitingRoomComponent EventHub Call', () => {
         tick();
         flushMicrotasks();
 
-        expect(videoCallService.stopScreenShare).toHaveBeenCalledTimes(1);
+        expect(videoCallService.stopScreenShare).toHaveBeenCalled();
```

Comment:
I think we should have the times called here. Was it just incrementing the call with each test? You could do

`videoCallService.stopScreenShare.calls.reset();`

in a beforeEach()